### PR TITLE
fix: add Arti Manputra

### PR DESCRIPTION
**Pull request by artimanputra**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot_8

## Note to reviewers 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/873"></a>

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
Thanks for your contribution 💥. In your PR seems to be an issue with the lockfile. Could you please revert the file in your PR 👍 Let me know if you need help with that.

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
Thanks for your contribution 💥. In your PR seems to be an issue with the lockfile. Could you please revert the file in your PR 👍 Let me know if you need help with that.

**Comment by artimanputra** [comment]
> Thanks for your contribution 💥. In your PR seems to be an issue with the lockfile. Could you please revert the file in your PR 👍 Let me know if you need help with that.

Yup, actually I am not getting what changes I have to do in that lockfile. Is I have to remove that last line?

**Comment by Cahllagerfeld** [comment]
The PR removes about 25000 lines in the `package-lock.json` You dont need to do any changes in that file, within your PR. You can just revert the changes in that file 👍

**Comment by Cahllagerfeld** [comment]
Do you need help with that?

**Comment by artimanputra** [comment]
> Do you need help with that?

Yes, I don't know how I can revert a single file. Can you please guide me?

File: public/data/artimanputra.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "url": "https://github.com/artimanputra",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ArtiManputra",
+        "icon": "twitter"
+      },
+      {
+        "name": "Check out my portfolio",
```
Comment (syedsohan) [inline public/data/artimanputra.json:18]:
```suggestion
        "name": "Check out my Portfolio",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/artimanputra.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "url": "https://jovial-stonebraker-c75f22.netlify.app/",
+        "icon": "globe"
+      },
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/arti-manputra-21183420a/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Read my blog",
```
Comment (syedsohan) [inline public/data/artimanputra.json:28]:
```suggestion
        "name": "Read my Blog",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - fix: fix: add Arti Manputra

**Comment by Cahllagerfeld** [comment]
I reverted the changes from your `package-lock.json`. Reverting is going back in the history of the file, not deleting. What I techinally did was setting the content of the file back to the state before your commit, so it doesnt get changed.

**Comment by artimanputra** [comment]
> I reverted the changes from your `package-lock.json`. Reverting is going back in the history of the file, not deleting. What I techinally did was setting the content of the file back to the state before your commit, so it doesnt get changed.

Thank You so much. I got this now.

**Review by syedsohan — APPROVED** [review]

**Comment by Cahllagerfeld** [comment]
I just fix the prettier issue. Thats a misconfiguration in the repo we are currently discussing in #868 if you're interested in that. When the action is finished, I'll have a look at it again, but I think its ready for merge now 👍

**Review by syedsohan — APPROVED** [review]

**Review by syedsohan — APPROVED** [review]

**Review by Cahllagerfeld — APPROVED** [review]
Looks good to me 💥

**Pull request merged**